### ISSUE TITLE
Inconsistent dates on CodaLab & Task CV homepage

### ISSUE THREAD

**Issue by stes** [VisionLearningGroup/visda-2018-public#17]
According to [CodaLab](https://competitions.codalab.org/competitions/18892#phases), the challenge ends on 22th August.
According to the [Task CV](https://sites.google.com/view/task-cv2018/dates-submission?authuser=0) homepage, the submission deadline is on 15th August.

Will the submission deadline be shifted back as well or is it mandatory to submit the paper on 15th?

**Comment by xcpeng** [comment]
Sorry for the confusion, the challenge will end on 22th August. There is no need to submit the abstract on 15th, you can submit after the challenge.

**Comment by lianqing11** [comment]
Will the classification deadline also extend to 28th Aug?

**Comment by xcpeng** [comment]
Yes, the classification deadline is extended to Aug. 28, 2018, 5 a.m. UTC/ Aug. 27 11:59 p.m. EST. This is the final deadline.
